### ISSUE TITLE
Replace scoring mechanism with binary classifier and threshold

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#29]
Right now scoring is done based off the SQRX classifier softmax output. Instead, train a net on the backend that goes directly to a binary classifier.

**Comment by wingman-jr-addon** [comment]
Ok, ended up closing this in #30 
The result is now the SQRXR family of models. The output is a single score using a loss function optimizing for AUC ROC, which should mapped against the ROC curve for proposed outputs. Current ROC curve is attached.
![sqrxr_62_roc](https://user-images.githubusercontent.com/55339824/72686685-e34da280-3abc-11ea-999e-ba02b9e42058.png)

The user-facing aspects of thresholding will be done in another issue.
